modules/integration_azure-storage-account/conf/01-count.yaml

```diff
@@ -0,0 +1,16 @@
+module: "Azure Storage Account"
+name: "count"
+filtering: "filter('resource_type', 'Microsoft.Storage/storageAccounts') and filter('primary_aggregation_type', 'true')"
+aggregation: ".fill(None, duration='1d').count(by=['subscription_id'])"
```

Comment:
why not simply `fill()` ?

Reply:
Because I want it to stop after one day :)

Reply:
ok, it is not common so I would to be sure :p